tests/integration/Core/Framework/Api/Controller/InfoControllerTest.php

```diff
@@ -498,6 +501,17 @@ public function testBaseAdminPaths(): void
         $config = json_decode($content, true, 512, \JSON_THROW_ON_ERROR);
         static::assertCount(3, $config['bundles']);
```

Comment:
```suggestion
        static::assertCount(5, $config['bundles']);
```

as you added two more bundles, this should be five, right?  

nevertheless you need to check this test, as it is failing

Reply:
Thank you for checking back. Reasonable question. It should be 4 as InfoController::getBundles filters one out with this check:
```php
            if (empty($styles) && empty($scripts) && $baseUrl === null) {
                continue;
            }
```